clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/description/DeployOpenstackAtomicOperationDescription.groovy

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2016 Veritas Technologies LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package com.netflix.spinnaker.clouddriver.openstack.deploy.description
+
+import com.netflix.spinnaker.clouddriver.deploy.DeployDescription
+import groovy.transform.AutoClone
+import groovy.transform.Canonical
+
+@AutoClone
+@Canonical
+class DeployOpenstackAtomicOperationDescription extends OpenstackAtomicOperationDescription implements DeployDescription {
+  String stackName
```

Comment:
You don't seem to be using the typical Spinnaker naming convention here. The other providers all use application/stack/freeFormDetails and a sequential numbering scheme to build a new server group name. This naming convention is exploited to properly group server groups in the ui and elsewhere.

e.g. aws: https://github.com/spinnaker/clouddriver/blob/master/clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/description/BasicAmazonDeployDescription.groovy#L27-L30
google: https://github.com/spinnaker/clouddriver/blob/master/clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/description/BasicGoogleDeployDescription.groovy#L28-L30
kubernetes: https://github.com/spinnaker/clouddriver/blob/master/clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/description/servergroup/DeployKubernetesAtomicOperationDescription.groovy#L27-L29